### ISSUE TITLE
Epic: Further automate season rollover resets

### ISSUE THREAD

**Issue by stenington** [Iridescent-CM/technovation-app#2250]
From #2229, we discovered a handful of things that had to be done manually as users began registering for a new season:

* update configured dates to be current for the upcoming season, or tests fail
* copy last season's certificates folder for the current season, or tests fail
  * but don't forget to eventually update those templates, or dates may be wrong
* cached `onboarded` values need to be reset
* training completion dates need to be reset

Automatic actions on season rollover:
* consent waivers need to be reset (currently through a scheduled job)
  * **NOT** configured on staging, which it should be
* parental consents need to be reset (they include seasonality in the code base and become voided when not in the current season)

It would be nice to automate more of these things to reduce the chance for bugs when going from season to season.

**Comment by hellafitz** [comment]
Next steps: 
- create a document listing work areas, current status and potential improvements (like Expected Developer Support Tasks doc for TG)
- prioritize tasks
- execute tasks

**Comment by stenington** [comment]
We have a scheduled task that runs daily but only actually does anything on the day the season rolls over. On that day, it voids all non-voided `ConsentWaiver`s. It looks like I wrote an alternative `reset_for_season` task that will also reset onboarded values and training completion dates, if we swapped it in to the scheduler. I don't actually remember writing it, but I think that would actually automate 2 of 4 things on the list.
